service/gradle/dependencies.gradle

```diff
@@ -20,7 +20,10 @@ dependencies {
 
   // Azure dependencies
   implementation 'com.azure:azure-storage-blob:12.21.1'
-  implementation group: 'com.azure.resourcemanager', name: 'azure-resourcemanager-monitor', version: '2.24.0'
+  implementation group: 'com.azure.resourcemanager', name: 'azure-resourcemanager-monitor', version: '2.27.0'
```

Comment:
Just curious, are these Azure deps in CRL? I think in general WSM gets its `azure-resourcemanager-*` dependencies transitively from CRL.